tests/tests_quizyapp/tests_views.py

```diff
@@ -0,0 +1,158 @@
+from django.test import TestCase, Client, RequestFactory
+from django.contrib.auth.models import User
+from django.urls import reverse
+import responses
+
+from quizyapp.test_mocks import mock_amount_2, mock_default, mock_category, mock_art_category
+from quizyapp.views import quiz, quiz_params
+# Create your tests here.
+
+
+class QuizViewTests(TestCase):
+    def setUp(self) -> None:
+        self.client = Client()
+        self.factory = RequestFactory()
+        self.user = User.objects.create_user(
+            username='oskar', email='oskar@example.com', password='top_secret')
+        responses.add(**mock_category)
+        responses.add(**mock_default)
+        responses.add(**mock_amount_2)
+        responses.add(**mock_art_category)
+        return super().setUp()
+
+    def test_quizy_url_returns_200_for_authorised_user(self):
+        request = self.factory.get(path=reverse('quiz'))
+        request.user = self.user
+        request.session = {}
+        response = quiz(request)
+
+        self.assertEqual(response.status_code, 200)
+
+    @responses.activate
+    def test_quizy_url_returns_302_for_unauthorised_user(self):
+
+        response = self.client.get(reverse('quiz'))
+        self.assertEqual(response.status_code, 302)
```

Comment:
Jeśli interesuje Cię przekierowanie pod określony URL, to możnaby to tu sprawdzić.